content/kubermatic/main/architecture/compatibility/supported-versions/_index.en.md

```diff
@@ -28,28 +28,21 @@ these migrations.
 In the following table you can find the supported Kubernetes versions for the
 current KKP version.
 
-| KKP version          | 1.29 | 1.28 | 1.27 | 1.26[^2] | 1.25[^2] | 1.24[^2] |
-| -------------------- | ---- | ---- | ---- | -------- | -------- | -------- |
-| 2.25.x               | ✓    | ✓    | ✓    | --       | --       | --       |
-| 2.24.x               | --   | ✓    | ✓    | ✓        | --       | --       |
-| 2.23.x               | --   | --   | ✓    | ✓        | ✓        | ✓        |
+| KKP version          | 1.31 | 1.30 | 1.29 | 1.28 | 1.27[^2] | 1.26[^2] |
+| -------------------- | -----| ---- | ---- | ---- | ---------| -------- |
+| 2.27.x               | ✓    |  ✓   | ✓    | ✓    | --       | --       |
```

Comment:
I don't think 2.27 will support 1.28.

Reply:
this is just WIP so that we remember there is a line for 2.27, I just duplicated 2.26. I would prefer if we didn't update this now but remembered on the KKP PRs to update things in docs at the same time.